app/services/file_thawer.py

```
@@ -52,7 +52,9 @@ def _finalize_staged_move(
             os.utime(str(final_destination), ns=(stat_info.st_atime_ns, stat_info.st_mtime_ns))
             source.unlink()
         except Exception as e:
-            logger.error("Failed to copy metadata or remove source in staged thaw finalization: %s", e)
+            logger.error(
+                "Failed to copy metadata or remove source in staged thaw finalization: %s", e
+            )
```

Ruff (0.15.14)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)